lib.php

```diff
@@ -24,10 +24,15 @@
 defined('MOODLE_INTERNAL') || die();
 
 /**
- * Callback function - injects Annoto's JS into every page.
+ * Allow plugins to provide some content to be rendered in the navbar.
+ * The plugin must define a PLUGIN_render_navbar_output function that returns
+ * the HTML they wish to add to the navbar.
+ *
+ * @return string HTML for the navbar
  */
-function local_annoto_before_footer() {
```

Comment:
@ilyaDymshits Shouldn't this be reverted back to `local_annoto_before_footer` or something? or is there timing issue related to kaltura?

Reply:
needs to check it. according to the moodle documentation if we wont to call js on <head> we should use requare->js('script', true) with second parameter true, but for some reason, the page renderer is falling down.